src/Connect.js

```diff
@@ -434,6 +416,7 @@ class Connect {
    * Accessor methods for Connect state.  The state consists of the key-value pairs below
    *  (did, doc, mnid, address, keypair, pushToken, and publicEncKey)
    */
+  get state ()        { return this._state }
```

Comment:
Should this return a new object so that state doesn't get mutated unitentionally? i.e. `return { ...this._state }`

Reply:
yeah it could